src/OpenTelemetry.Exporter.Geneva/Internal/Tld/TldLogCommon.cs

```diff
@@ -0,0 +1,232 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+using Microsoft.Extensions.Logging;
+using OpenTelemetry.Logs;
+
+namespace OpenTelemetry.Exporter.Geneva.Tld;
+
+internal abstract class TldLogCommon : IDisposable
```

Comment:
what's the purpose of the new Common class?
I only see one class (TldLogExporter) inheriting this.